### ISSUE TITLE
Process crashed when node-red started on Cisco IOx

### ISSUE THREAD

**Issue by Shadow-Ding** [node-red/node-red-docker#126]
### What are the steps to reproduce?
I think it can not be easily reproduced becaused it was running on a certain platform called Cisco IOx.
### What happens?
I make the docker image from the docker-hub `nodered/node-red-docker`. It works well on my dev device (Ubuntu 14.04 x86_64 as Docker host), as below:
```
~/node_modules/node-red $ npm start

> node-red@0.20.7 start /usr/src/node-red/node_modules/node-red
> node red.js

24 Aug 03:00:30 - [info] 

Welcome to Node-RED
===================

24 Aug 03:00:30 - [info] Node-RED version: v0.20.7
24 Aug 03:00:30 - [info] Node.js  version: v8.16.0
24 Aug 03:00:30 - [info] Linux 4.4.0-31-generic x64 LE
24 Aug 03:00:31 - [info] Loading palette nodes
24 Aug 03:00:31 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
24 Aug 03:00:31 - [warn] rpi-gpio : Cannot find Pi RPi.GPIO python library
24 Aug 03:00:32 - [info] Dashboard version 2.16.1 started at /ui
24 Aug 03:00:33 - [info] Settings file  : /usr/src/node-red/.node-red/settings.js
24 Aug 03:00:33 - [info] Context store  : 'default' [module=memory]
24 Aug 03:00:33 - [info] User directory : /usr/src/node-red/.node-red
24 Aug 03:00:33 - [warn] Projects disabled : editorTheme.projects.enabled=false
24 Aug 03:00:33 - [info] Flows file     : /usr/src/node-red/.node-red/flows_53e61fd110b5.json
24 Aug 03:00:33 - [info] Creating new flow file
24 Aug 03:00:33 - [warn] 

---------------------------------------------------------------------
Your flow credentials file is encrypted using a system-generated key.

If the system-generated key is lost for any reason, your credentials
file will not be recoverable, you will have to delete it and re-enter
your credentials.

You should set your own key using the 'credentialSecret' option in
your settings file. Node-RED will then re-encrypt your credentials
file using your chosen key the next time you deploy a change.
---------------------------------------------------------------------

24 Aug 03:00:33 - [info] Starting flows
24 Aug 03:00:33 - [info] Started flows
```

Then, I want to deploy it on a Cisco device using Cisco IOx, I use Cisco development tool to package this Docker image. When I start node-red on this device, it crashed as below:
```
~ $ npm start

> node-red-docker@1.0.0 start /usr/src/node-red
> node $NODE_OPTIONS node_modules/node-red/red.js -v $FLOWS

24 Aug 06:02:23 - [info] 

Welcome to Node-RED
===================

24 Aug 06:02:23 - [info] Node-RED version: v0.20.7
24 Aug 06:02:23 - [info] Node.js  version: v8.16.0
24 Aug 06:02:23 - [info] Linux 4.1.46-rt51-yocto-standard x64 LE
24 Aug 06:02:24 - [info] Loading palette nodes
24 Aug 06:02:25 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
24 Aug 06:02:25 - [warn] rpi-gpio : Cannot find Pi RPi.GPIO python library
Killed
~ $ node node_modules/node-red/red.js 
24 Aug 06:04:18 - [info] 

Welcome to Node-RED
===================

24 Aug 06:04:18 - [info] Node-RED version: v0.20.7
24 Aug 06:04:18 - [info] Node.js  version: v8.16.0
24 Aug 06:04:18 - [info] Linux 4.1.46-rt51-yocto-standard x64 LE
24 Aug 06:04:19 - [info] Loading palette nodes
24 Aug 06:04:21 - [warn] rpi-gpio : Raspberry Pi specific node set inactive
24 Aug 06:04:21 - [warn] rpi-gpio : Cannot find Pi RPi.GPIO python library
24 Aug 06:04:24 - [info] Dashboard version 2.16.1 started at /ui
Killed
```
### What do you expect to happen?
As I know, when I use Cisco development tool to package the docker image, it will change the some file in the image. So I want to know what may cause this problem? 
### Please tell us about your environment:

- [x] Node-RED version: v0.20.7
- [x] node.js version: v8.16.0
- [x] npm version: v6.4.1
- [x] Platform/OS: Cisco IOx/Linux 4.1.46-rt51-yocto-standard x64 LE
- [ ] Browser: 

Thanks

**Comment by RaymondMouthaan** [comment]
As per #127, native GPIO support has been removed, so it wouldn't crash no longer on this platform. 

Can be closed, once #127 is merged.

**Comment by dceejay** [comment]
Node-RED 1.0.0 is now available on docker hub - https://hub.docker.com/r/nodered/node-red.

This should now close this issue as the GPIO nodes are no longer included by default.
 
There are a few changes to the new release, please read the README for further details.
